src/main/java/it/eng/idsa/businesslogic/configuration/SSLContextParametersConfiguration.java

```diff
@@ -27,12 +27,15 @@
 	@Value("${application.keyStorePassword}")
 	private String keyStorePassword;
 	
-	@Value("${application.trustStoreName}")
+	@Value("${application.trustStoreName:}")
 	private String trustStoreName;
 	
 	@Value("${application.trustStorePassword}")
 	private String trustStorePassword;
 	
+	@Value("${application.idscp2.isEnabled}")
+	private boolean idscp2IsEnabled;
+	
 	
 	
 	@Bean
```

Comment:
Can you check if this bean can be conditional, based on idscp2.isEnabled, if connector will start in both cases correct - idscp2 = true and idscp2 = false?
Something like this:
https://reflectoring.io/spring-boot-conditionals/
So that we can leave logic as-is and create bean only when idscp2 is enabled.